examples/python_logo.qml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Sure - looks like I have been starring into code for a bit too ling in this case. :)